packages/shared/src/components/fields/form/FormWrapper.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement, ReactNode } from 'react';
+import classNames from 'classnames';
+import { Button, ButtonVariant } from '../../buttons/Button';
+
+interface Copy {
+  left?: string;
+  right?: string;
+}
+
+interface FormWrapperProps {
+  children: ReactNode;
+  className?: string;
+  form: string;
+  copy?: Copy;
+  onLeftClick?(): void;
+  onRightClick?(): void;
+}
+
+export function FormWrapper({
+  children,
+  className,
+  form,
+  copy = {},
+  onLeftClick,
+  onRightClick,
+}: FormWrapperProps): ReactElement {
+  return (
+    <div className={classNames('flex flex-col', className)}>
+      <div className="flex flex-row justify-between border-b border-theme-divider-tertiary px-4 py-2">
+        <Button variant={ButtonVariant.Tertiary} onClick={onLeftClick}>
```

Comment:
These are always solid Tertiary? was wondering if it would be easier to simply accept left and right buttons as complete elements?

Reply:
If we do that then I don't even think we need a wrapper in the first place, just styled components.

If we're going to be using this the same way in a couple of places I think it doesn't hurt to have it non customisable at the moment, we can customise later base on our use cases.